common/delete-link.js

```diff
@@ -47,12 +48,14 @@
             link: function(scope) {
                 scope.deleteFn = function deleteFn() {
                     if (!CONFIRM_DELETE) {
+                        // TODO: Wrap callback in catcher fn
```

Comment:
Presumably this `callback` function would have already been wrapped by the `catchWrapper()`.